src/transformation/utils/annotations.ts

```diff
@@ -118,7 +117,7 @@ export function isTupleReturnCall(context: TransformationContext, node: ts.Node)
     const signature = context.checker.getResolvedSignature(node);
     if (signature) {
         if (getSignatureAnnotations(context, signature).has(AnnotationKind.TupleReturn)) {
-            context.diagnostics.push(annotationDeprecated(node, AnnotationKind.TupleReturn));
+            context.diagnostics.push(annotationRemoved(node, AnnotationKind.TupleReturn));
```

Comment:
I think diagnostic.push should be removed from isTupleReturnCall(). isTupleReturnCall should just return true or false without side effects.
The diagnostic should only be pushed at the call location of isTupleReturnCall().
e.g.
```
if (isTupleReturnCall()) {
    context.diagnostic.push...
}
```
Which seems to already be the case???? https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1090/files#diff-877ef39cd04cebdd5a0baf7a2090e119498765d5afc71abe08f32422d1dd1984R252
I wonder why the test does not fail because with this implementation, the diagnostic should get pushed multiple times?